client/shipping-label/state/reducer.js

```diff
@@ -422,21 +422,24 @@ reducers[ SET_PACKAGE_TYPE ] = ( state, { packageId, boxTypeId } ) => {
 	const newPackages = { ...state.form.packages.selected };
 	const oldPackage = newPackages[ packageId ];
 
+	const oldBox = state.form.packages.all[ oldPackage.box_id ];
+	const newBox = state.form.packages.all[ boxTypeId ];
+	const weight =
```

Comment:
@v18 No idea what minimum precision we need but let's say we can agree on some limit `digits` that's less than 10. Will `+arithmeticResult.toFixed( digits )` be good enough?

Test case demonstrating the issue independent of this PR (i.e. in `master`): put a weight 3 item and a weight 0.3 item in the same package, and then take out the weight 3 item — the Total Weight winds up as 0.2999999999999998